packages/suite-desktop/src-electron/libs/logger.ts

```diff
@@ -38,6 +38,14 @@ class Logger implements ILogger {
         };
 
         if (this.logLevel > 0 && this.options.writeToDisk) {
+            if (!this.options.outputPath || !this.options.outputFile) {
+                this.error(
+                    'logger',
+                    "Can't write log to file because outputFile or outputPath are invalid",
```

Comment:
maybe add the `options` object to this message. to be able to track which one is invalid?

Reply:
Addressed in 08966ef (squashed).